osidb_bindings/bindings/python_client/api/osidb/__init__.py

```diff
@@ -74,3 +74,82 @@
+    "osidb_api_v1_affects_cvss_scores_retrieve",
+    "osidb_api_v1_affects_cvss_scores_update",
+    "osidb_api_v1_affects_cvss_scores_destroy",
+    "osidb_api_v1_affects_retrieve",
+    "osidb_api_v1_affects_update",
+    "osidb_api_v1_affects_destroy",
+    "osidb_api_v1_affects_bulk_update",
+    "osidb_api_v1_affects_bulk_create",
+    "osidb_api_v1_affects_bulk_destroy",
+    "osidb_api_v1_alerts_list",
+    "osidb_api_v1_alerts_retrieve",
+    "osidb_api_v1_audit_list",
+    "osidb_api_v1_audit_retrieve",
+    "osidb_api_v1_audit_update",
+    "osidb_api_v1_available_flaws_retrieve",
+    "osidb_api_v1_flaws_list",
+    "osidb_api_v1_flaws_create",
+    "osidb_api_v1_flaws_acknowledgments_list",
+    "osidb_api_v1_flaws_acknowledgments_create",
+    "osidb_api_v1_flaws_acknowledgments_retrieve",
+    "osidb_api_v1_flaws_acknowledgments_update",
+    "osidb_api_v1_flaws_acknowledgments_destroy",
+    "osidb_api_v1_flaws_comments_list",
+    "osidb_api_v1_flaws_comments_create",
+    "osidb_api_v1_flaws_comments_retrieve",
+    "osidb_api_v1_flaws_cvss_scores_list",
+    "osidb_api_v1_flaws_cvss_scores_create",
+    "osidb_api_v1_flaws_cvss_scores_retrieve",
+    "osidb_api_v1_flaws_cvss_scores_update",
+    "osidb_api_v1_flaws_cvss_scores_destroy",
+    "osidb_api_v1_flaws_labels_list",
+    "osidb_api_v1_flaws_labels_create",
+    "osidb_api_v1_flaws_labels_retrieve",
+    "osidb_api_v1_flaws_labels_update",
+    "osidb_api_v1_flaws_labels_destroy",
+    "osidb_api_v1_flaws_package_versions_list",
+    "osidb_api_v1_flaws_package_versions_create",
+    "osidb_api_v1_flaws_package_versions_retrieve",
+    "osidb_api_v1_flaws_package_versions_update",
+    "osidb_api_v1_flaws_package_versions_destroy",
+    "osidb_api_v1_flaws_promote_create",
+    "osidb_api_v1_flaws_references_list",
+    "osidb_api_v1_flaws_references_create",
+    "osidb_api_v1_flaws_references_retrieve",
+    "osidb_api_v1_flaws_references_update",
+    "osidb_api_v1_flaws_references_destroy",
+    "osidb_api_v1_flaws_reject_create",
+    "osidb_api_v1_flaws_retrieve",
+    "osidb_api_v1_flaws_update",
+    "osidb_api_v1_labels_list",
+    "osidb_api_v1_labels_retrieve",
+    "osidb_api_v1_manifest_retrieve",
+    "osidb_api_v1_schema_retrieve",
+    "osidb_api_v1_status_retrieve",
+    "osidb_api_v1_trackers_list",
+    "osidb_api_v1_trackers_create",
+    "osidb_api_v1_trackers_retrieve",
+    "osidb_api_v1_trackers_update",
+    "osidb_api_v2beta_affects_cvss_scores_list",
+    "osidb_api_v2beta_affects_cvss_scores_create",
+    "osidb_api_v2beta_affects_cvss_scores_retrieve",
+    "osidb_api_v2beta_affects_cvss_scores_update",
+    "osidb_api_v2beta_affects_cvss_scores_destroy",
+    "osidb_api_v2beta_flaws_cvss_scores_list",
+    "osidb_api_v2beta_flaws_cvss_scores_create",
+    "osidb_api_v2beta_flaws_cvss_scores_retrieve",
+    "osidb_api_v2beta_flaws_cvss_scores_update",
+    "osidb_api_v2beta_flaws_cvss_scores_destroy",
```

Comment:
shouldn't this be v2?

Reply:
I am using API schema for OSIDB 4.16.0 at the moment. Once OSIDB 5.0.0 is fully release, I'll release with the new schema file.